torch/_inductor/runtime/runtime_utils.py

```diff
@@ -118,10 +119,29 @@ def load_triton():
     return triton_do_bench(*args, **kwargs)[0]
 
 
-def do_bench_cpu(fn, warmup=5, times=20):
+def do_bench_cpu(fn, warmup=5, times=20, min_duration=0.1):
```

Comment:
the triton / cuda do_bench takes in a duration to do benchmarking for and from that calculates # of warmup and iterations. is there a reason to invert it here ?

Reply:
Thanks for the suggestion. Revised according to triton's definition.